### ISSUE TITLE
'orbitize/_kepler.pyx' doesn't match any files

### ISSUE THREAD

**Issue by ulrich-hiller** [sblunt/orbitize#81]
I am on an CentOs 7 machine  with the most recent anaconda. numpy and cathon are upgraded.
Installing orbitize (no matter whther fron pip or source) gives the following error:

pip install orbitize
Collecting orbitize
  Downloading https://files.pythonhosted.org/packages/32/14/e0da52c27d509cea675794d41cbc6b252c5ea3953d8a9427b386baac04cb/orbitize-1.0.0.tar.gz (76kB)
    100% |████████████████████████████████| 81kB 10.3MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-cwMWKA/orbitize/setup.py", line 41, in <module>
        ext_modules=get_extensions(),
      File "/tmp/pip-install-cwMWKA/orbitize/setup.py", line 29, in get_extensions
        extensions = cythonize([Extension("orbitize._kepler", ["orbitize/_kepler.pyx"])])
      File "/usr/lib64/python2.7/site-packages/Cython/Build/Dependencies.py", line 956, in cythonize
        aliases=aliases)
      File "/usr/lib64/python2.7/site-packages/Cython/Build/Dependencies.py", line 801, in create_extension_list
        for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
      File "/usr/lib64/python2.7/site-packages/Cython/Build/Dependencies.py", line 111, in nonempty
        raise ValueError(error_msg)
    ValueError: 'orbitize/_kepler.pyx' doesn't match any files

What could be the reason?
Mny thanks in advance for the help. 
-ulrich

**Comment by sblunt** [comment]
Thanks for the bug report, Ulrich! Our team was excited to respond to our first "real" issue. :) 

The problem should be fixed in the version I just released (version 1.0.1). Try pip installing again and let me know what happens.

For posterity, the problem was that .pyx and .c files were not included in the .tar.gz distribution file. These files are needed to build `orbitize` on some machines. This was fixed by adding a `MANIFEST.in` file.
